ArduCopter/mode_acro.cpp

```diff
@@ -58,7 +66,7 @@ void ModeAcro::run()
     }
 
     // output pilot's throttle without angle boost
-    attitude_control->set_throttle_out(get_pilot_desired_throttle(),
+    attitude_control->set_throttle_out(limit_throttle_out ? 0 : get_pilot_desired_throttle(),
```

Comment:
What does this fix? Its a little odd. Setting throttle should be OK and get ignored in motors.